devices/bin/__init__.py

```diff
@@ -0,0 +1,66 @@
+# Copyright (C) 2020 Denis Gavrilyuk <karpa4o4@gmail.com>
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program. If not, see <http://www.gnu.org/licenses/>.
+
+""""""
```

Comment:
@eugulixes, can you suggest a name for the module?

Reply:
`Module containing the interface for serving the devices supported by Pieman to clients.`